checker/votour.ml

```diff
@@ -20,14 +19,14 @@ let rec read_num max =
   if l = "u" then None
   else if l = "x" then quit ()
   else
-    try
-      let v = int_of_string l in
+    match int_of_string l with
+    | v ->
       if v < 0 || v >= max then
         let () =
           Printf.printf "Out-of-range input! (only %d children)\n%!" max in
         read_num max
       else Some v
-    with Failure "int_of_string" ->
+    | exception Failure _ ->
```

Comment:
Ocaml makes no guarantee that the string will stay the same in the future, and the code ensures the exception can only come from int_of_string.
I could change `exception Failure _` to `exception _` if you prefer.